server/repository/migrations/postgres/2022-06-07T17-00_create_encounter_table/up.sql

```diff
@@ -12,5 +12,6 @@ CREATE TABLE encounter (
     program TEXT NOT NULL,
     start_datetime TIMESTAMP NOT NULL,
     end_datetime TIMESTAMP,
-    status encounter_status
+    status encounter_status,
```

Comment:
no migration scripts yet..?
I have only just recreated my database to test a previous PR and the sync takes ages.. so I've just created the column manually 😉

Reply:
Incidentally, why does the `[clinician]` table have a `store_id` when there is also a `[clinician_store_join]` table?

Reply:
I am working with a small DB which makes sync quite fast, which DB are you working with?

> Incidentally, why does the [clinician] table have a store_id when there is also a [clinician_store_join] table?

its coming from mSupply, I could remove the clinician store_id because we don't make use of it in omSupply... Will make a change

Reply:
I removed the store_id...